src/llm_vm/completion/data_synthesis.py

```diff
@@ -37,10 +37,10 @@ def data_synthesis(self, optimizer, prompt, response, openai_key=None, completio
             return list(pickle.load(new_file))
         datapoints = []
         final_prompt = '{"prompt": "' +prompt+'"  , "response": "' +response+'" }'
-        final_prompt = "Generate 1 json similar to the one below. \n" + final_prompt
+        final_prompt = "Generate 1 json similar to the one below. \n" + json.dumps(final_prompt)
```

Comment:
What is the motivation behind doing this?

Reply:
I had done this to ensure that the data that is generated has a particular format which will make processesing easier